applications/conversations/models/class.conversationsmodel.php

```diff
@@ -67,16 +67,21 @@ public function checkForSpam($type, $skipSpamCheck = false) {
      * @param bool $idsOnly The returns only the userIDs or everything from UserConversation.
      * @param bool $limit
      * @param bool $offset
+     * @param bool|null $active **true** for active participants, **false** for users who have left the conversation and **null** for everyone.
      *
      * @return array Array of users or userIDs depending on $idsOnly's value.
      */
-    public function getConversationMembers($conversationID, $idsOnly = true, $limit = false, $offset = false) {
+    public function getConversationMembers($conversationID, $idsOnly = true, $limit = false, $offset = false, $active = null) {
         $conversationMembers = [];
 
         $userConversation = new Gdn_Model('UserConversation');
-        $userMembers = $userConversation->getWhere([
-            'ConversationID' => $conversationID
-        ], 'UserID', 'asc', $limit, $offset)->resultArray();
+        $where = ['ConversationID' => $conversationID];
+        if ($active === true) {
+            $where['Deleted'] = 0;
+        } elseif ($active === false) {
+            $where['Deleted >'] = 0;
```

Comment:
I know it works but this is implying that Deleted **COULD BE** different than 1 or 0 which would be anormal.
We should put a note highlighting that or use `$where['Deleted'] = 1;`.
The latter is also more resilient to changes, if we decide to make Deleted = 2 a thing. (Doesn't make sense to present me though.)

FYI: In general GT is also bad for performance but since we are already filtering by conversation it is not a real concern in this case.